### DISCUSSION TITLE
Calculating Min/Max when Same Code is Multiple Rows

### DISCUSSION THREAD

**Discussion by foustsWCH** [CMSgov/hospital-price-transparency#46] [Q&A] [answered]
I want to make sure that I am displaying min/max correctly in my file.  In looking at CSV wide examples, I see two different scenarios where the same code has multiple rows due to additional qualifiers. However, it seems like how min/max rates are handled differs between these two scenarios and I'm not sure why.  What is the difference between these scenarios that would make the min/max be considered differently?

Example 1: There is a revenue code example where revenue code 762 has multiple rows because a specific payer has different rates depending on whether there is "rule out myocardial infarction" or not.
- In this scenario, the min/max listed appears to be the overall min/max for the revenue code.  It is the same on both rows.
<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

description | code\|1 | code\|1\|type | standard_charge\|Platform_Health_Insurance\|PPO\|negotiated_dollar | standard_charge\|Platform_Health_Insurance\|PPO\|methodology | additional_payer_notes\|Platform_Health_Insurance\|PPO | standard_charge\|Region_Health_Insurance\|HMO\|negotiated_dollar | standard_charge\|Region_Health_Insurance\|HMO\|methodology | additional_payer_notes\|Region_Health_Insurance\|HMO | standard_charge\|min | standard_charge\|max
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Treatment or observation room —   observation room | 762 | RC | 8000 | case rate | Negotiated standard charge without surgery and without rule out   myocardial infarction | 9000 | case rate |   | 8000 | 10000
Treatment or observation room —   observation room | 762 | RC | 10000 | case rate | Negotiated standard charge without   surgery and with rule out myocardial infarction |||| 8000 | 10000

</body>

</html>

Example 2: There is a "per diem" example where HCPCS H0017 has multiple rows because a specific payer has different rates depending on what day of the stay it is.  
- The min/max appear to be calculated at the "row" level, based on each circumstance, not for the code overall.  This seems like it is initutively wrong.  
 For instance, for "Behavioral health; residential (hospital residential treatment program), without room and board, per diem, days 1-3" the min and max are both listed as 2000.  However, Platform_Health_Insurance|PPO's per diem rate (for any day of the stay) is $1500, so shouldn't the min be $1500?  If it were day 1-3, Platform_Health_Insurance|PPO would be paying a per diem rate of $1500, which is lower than the $2000 that Region_Health_Insurance|HMO has negotiated.
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

description | code\|1 | code\|1\|type | standard_charge\|Platform_Health_Insurance\|PPO\|negotiated_dollar | standard_charge\|Platform_Health_Insurance\|PPO\|methodology | additional_payer_notes\|Platform_Health_Insurance\|PPO | standard_charge\|Region_Health_Insurance\|HMO\|negotiated_dollar | standard_charge\|Region_Health_Insurance\|HMO\|methodology | additional_payer_notes\|Region_Health_Insurance\|HMO | standard_charge\|min | standard_charge\|max
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Behavioral health; residential (hospital   residential treatment program), without room and board, per diem | H0017 | HCPCS | 1500 | per diem |   |   |   |   | 1500 | 1500
Behavioral health; residential (hospital   residential treatment program), without room and board, per diem, days 1-3 | H0017 | HCPCS |   |   |   | 2000 | per diem |   | 2000 | 2000
Behavioral health; residential (hospital   residential treatment program), without room and board, per diem, days 4-5 | H0017 | HCPCS |   |   |   | 1800 | per diem |   | 1800 | 1800
Behavioral health; residential (hospital   residential treatment program), without room and board, per diem, days 6+ | H0017 | HCPCS |   |   |   | 1200 | per diem |   | 1200 | 1200

</body>

</html>

I am assuming I'm missing some nuance in the data.  Can you help clarify the differences between the examples?

**Comment by RidenourK** [comment, marked as answer]
@foustsWCH. Thank you for your question. 

At 45 CFR 180.20 we defined the “de-identified minimum negotiated charge” to mean the lowest charge that a hospital has negotiated with all third-party payers **for an item or service** and the ‘‘de-identified maximum negotiated charge’’ to mean the highest charge that a hospital has negotiated with all third-party payers **for an item or service**. In the CMS Template(s) these data elements would be encoded using data from the Payer-specific Negotiated Charge: Dollar Amount data element.

**Reply by foustsWCH** [reply]
@RidenourK 
Thank you for citing the regulations.  I had reviewed the final rule in its entirety, as well as the examples provided on this site prior to asking my question and was still unclear.  After reviewing your response, I remain unclear.

Since you bolded "**for an item or service**", can you clarify whether "item or service" should be considered to mean each unique item or service defined _by the code(s) only_ or if additional qualifying context placed in other fields (either in the text description of the item or additional payer notes) differentiates different items or services that then need the min/max calculated separately?

I specifically used your own examples because they don't seem consistent (I may be misunderstanding the logic).   I would hope you would be able to provide a clarifying explanation of the logic used to construct these examples since they are your own examples and are provided with the intent to help hospitals understand compliant ways to report the data.

So, to use the examples in my original question (which come from your own CSV Wide template examples):
**Example 1:**
Option A: Is it correct to think that "Revenue Code 762" is its own "item or service" where the min/max reported needs to be the overall min/max for that code regardless of other context in the payer notes?
OR
Option B: Since one payer separates "Revenue Code 762 without surgery and without rule out myocardial infarction" and "Revenue Code 762 without surgery and with rule out myocardial infarction", thus creating two different scenarios, is it correct to consider those two situations to be two different "items or services" that need the min/max calculated separately for each context?

**Example 2:**
Option A: Is it correct to think that "HCPCS H0017" is its own "item or service" and that the min/max reported need to be the overall min/max for that code regardless of other factors?

Option B: Since one payer separates "HCPCS H0017" into "Hospital Residential Treatment Program days 1-3", "Hospital Residential Treatment Program days 4-6", and "Hospital Residential Treatment Program days 6+" to specify payment for three different scenarios, is it correct to consider those three situations three different "items or services" that need the min/max calculated separately for each context?

To me, it appears that in Example 1, your file calculates the min/max using approach A (i.e. treating "Revenue Code 762" as one item and calculating an overall min/max for the code regardless of the other context put in the additional payer notes on separate lines).  
It appears that in Example 2, your file calculates the mix/max using approach B (i.e. treating each scenario as its own distinct "item or service" and providing the min/max for each scenario separately.)

I don't understand the difference between the two scenarios, so am hoping for clarification.
